src/cli/cli.cpp

```diff
@@ -285,6 +285,20 @@ otError Interpreter::ProcessHelp(uint8_t aArgsLength, char *aArgs[])
     return OT_ERROR_NONE;
 }
 
+#if OPENTHREAD_CONFIG_BORDER_AGENT_ENABLE
+otError Interpreter::ProcessBorderAgent(uint8_t aArgsLength, char *aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    VerifyOrExit(aArgsLength == 1 && strcmp(aArgs[0], "port") == 0, error = OT_ERROR_INVALID_COMMAND);
```

Comment:
nit: add the `state` command as well. It could be helpful for debugging commissioning issues.

Reply:
Add it when we need it? This PR is just for using ephemeral port.

Reply:
It will be used when there are commissioning issues reported, such as https://github.com/openthread/ot-commissioner/issues/193#issuecomment-822137253. With command `ba state`, we can at least verify if the Border Agent is running. You may consider carry it in this PR or I can add it in https://github.com/openthread/openthread/pull/6481.

Reply:
Prefer adding it in the PR that is going to use it, which would also cover the code.